tests/openbb_terminal/core/log/collection/test_log_sender.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from openbb_terminal.core.log.collection import log_sender
+from openbb_terminal.core.log.generation.settings import (
+    Settings,
+    AppSettings,
+    AWSSettings,
+    LogSettings,
+)
+
+settings = Settings(
+    app_settings=AppSettings(
+        commit_hash="MOCK_COMMIT_HASH",
+        name="MOCK_COMMIT_HASH",
+        identifier="MOCK_COMMIT_HASH",
+        session_id="MOCK_SESSION_ID",
+    ),
+    aws_settings=AWSSettings(
+        aws_access_key_id="MOCK_AWS_ACCESS_KEY_ID",
+        aws_secret_access_key="MOCK_AWS",  # pragma: allowlist secret
+    ),
+    log_settings=LogSettings(
+        directory=Path("."),
```

Comment:
On my setup it is still generating and leaving some file.
Probably because of this Path.
Let's mock the path so it won't generate any actual file.